src/common/com/intellij/plugins/haxe/ide/folding/HaxeFoldingBuilder.java

```diff
@@ -445,6 +475,23 @@ public String getPlaceholderText(@NotNull ASTNode node) {
 
   @Override
   public boolean isCollapsedByDefault(@NotNull ASTNode node) {
+    final PsiElement psiElement = node.getPsi();
+
+    final CodeFoldingSettings settings = CodeFoldingSettings.getInstance();
+    //final HaxeFoldingSettings haxeSettings = HaxeFoldingSettings.getInstance();
+
+    if (psiElement instanceof HaxeImportStatement || psiElement instanceof HaxeUsingStatement) {
+      return settings.COLLAPSE_IMPORTS;
+    }
+
+    final IElementType elementType = node.getElementType();
+    if (isComment(elementType, node)) {
```

Comment:
Since this block is only working with document comments, you can probably use `isDocComment()` here and not retest it below.  Not a big deal, but it would be a bit cleaner.